R/Rules-methods.R

```diff
@@ -773,6 +773,59 @@ setMethod(
     }
 )
 
+
+# maxDose-IncrementsHSRBeta ----
+
+#' @rdname maxDose
+#'
+#' @description Determine the maximum possible dose for escalation.
+#'
+#' @aliases maxDose-IncrementsHSRBeta
+#' @example examples/Rules-method-maxDose-IncrementsHSRBeta.R
+#' @export
+setMethod(
+  "maxDose",
+  signature = signature(
+    increments = "IncrementsHSRBeta",
+    data = "Data"
+  ),
+  definition = function(increments, data, ...) {
+    # summary of observed data per dose level
```

Comment:
```suggestion
    # Summary of observed data per dose level.
```
please similar below (coding convention)